synapse/axon.py

```diff
@@ -451,6 +453,9 @@ def __init__(self, axondir, **opts):
 
         corepath = os.path.join(self.axondir, 'axon.db')
         self.core = s_cortex.openurl('sqlite:///%s' % corepath)
+        for name, model in axon_model.getBaseModels():
+            self.core.addDataModel(name, model)
```

Comment:
we want to load this as a configable module using setConfOpt (see https://github.com/vertexproject/synapse/blob/master/synapse/tests/test_cortex.py#L1720) instead of calling addDataModel directly.